tables/he-common.uti

```diff
@@ -0,0 +1,581 @@
+# liblouis: hebrew-common
+
+#-license: lgpl-2.1
+
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+# This is a common character file that supports Hebrew forward and
+# back translation. 
+#Authors: Matityahu Yeshurun, Paul Geoghegan, and Leonard De
+# Ruijter
+# Copyright 2024 Matt Yeater- Yeshurun and Paul Geoghegan
+This is a common character file that supports Hebrew forward and back translation. Matthew Yeater, Paul Geoghegan, and Leonard De Ruijter
```

Comment:
This line currently breaks the table, since it should be a comment and it isn't. Given there's a comment that reads something similar, I think it can be safely removed.

Reply:
Fixed